special-interest-groups/infrastructure/README.md

```diff
@@ -0,0 +1,22 @@
+
+# SIG Infrastructure and Indexing
+
+## Goals
+
+We build and maintain the community RPC endpoints and tracker.
+
+## Meetings
+
+TBD how meetings will be announced and planned starting Jan 2023. 
+
+## Projects
+
+- [icon-tracker](https://github.com/sudoblockio/icon-tracker)
+
+## Contributing
+
+Please file issues in the [icon-tracker](https://github.com/sudoblockio/icon-tracker) or any of the relevant associated repositories. 
+
+## Contact
+
+Rob Cannon (rob@sudoblock.io)
```

Comment:
Should include the tracker_dev discord channel, and possibly rename that channel to infrastructure_dev